tools/web3js-example/scripts/transferHbars.js

```diff
@@ -19,21 +19,25 @@
  */
 
 require('dotenv').config();
-const Web3 = require('web3');
-const Web3HttpProvider = require('web3-providers-http');
+const { Web3 } = require('web3');
 
 module.exports = async (amount = 100_000_000_000) => {
-  const web3 = new Web3(new Web3HttpProvider(process.env.RELAY_ENDPOINT));
+  const web3 = new Web3(new Web3.providers.HttpProvider(process.env.RELAY_ENDPOINT));
   const wallet = await web3.eth.accounts.wallet.add(process.env.OPERATOR_PRIVATE_KEY);
   const walletReceiver = await web3.eth.accounts.wallet.add(process.env.RECEIVER_PRIVATE_KEY);
 
-  console.log(`Balance before tx: ${await web3.eth.getBalance(walletReceiver.address)}`);
+  console.log(`Balance before tx: ${await web3.eth.getBalance(walletReceiver[0].address)}`);
   // Keep in mind that TINYBAR to WEIBAR coefficient is 10_000_000_000
-  const signedTx = await wallet.signTransaction({
-    'to': walletReceiver.address,
-    'value': amount,
-    'gas': 300000
-  });
-  await web3.eth.sendSignedTransaction(signedTx.rawTransaction);
-  console.log(`Balance after  tx: ${await web3.eth.getBalance(walletReceiver.address)}`);
-};
+  const gasPrice = await web3.eth.getGasPrice();
+  const transaction = {
+    from: wallet[0].address,
+    to: walletReceiver[1].address,
+    gas: 300000,
+    gasPrice: gasPrice,
+    value: amount, // value should be passed in wei. For easier use and to avoid mistakes we utilize the auxiliary `toWei` function.
+  };
+
+  await web3.eth.sendTransaction(transaction);
+  console.log(`Balance after  tx: ${await web3.eth.getBalance(walletReceiver[0].address)}`);
+  
+};
```

Comment:
blank line

Reply:
I thought we wanted the blank line?

Reply:
Removed.

Reply:
yeah, we wanted a blank line. I mean to say, blank line missing. sorry for the mess up😅